consoleApp/src/main/java/com/infoshare/service/VolunteerRepository.java

```diff
@@ -0,0 +1,74 @@
+package com.infoshare.service;
+
+import com.infoshare.domain.TypeOfHelp;
+import com.infoshare.domain.Volunteer;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+
+public class VolunteerRepository {
+
+    private static final String DB_FILE = "Volunteer.csv";
+
+    public void printFilteredList(){
+        Scanner scan = new Scanner(System.in);
+        System.out.println("Z jakiego miasta chcesz wyszukac wolontariuszy?");
+        String inputCity = scan.nextLine();
+        System.out.println("Jakiego typu pomocy szukasz?");
+        printTypesOfHelp();
+        String inputTypeOfHelp = scan.nextLine();
+        try {
+            List <Volunteer> availableVolunteers = readFromFile();
+            List <Volunteer> filteredList;
+            filteredList = availableVolunteers.stream().
+                    filter(Volunteer::isAvailable).
+                    filter(v -> v.getLocation().toLowerCase().equals(inputCity.toLowerCase())).
```

Comment:
Czy nie lepiej zastosowac equalsIgnoreCase() zamiast dwoch polecen equals i to LowerCase ?

Reply:
Nie próbowałem tak, jak znajdę czas żeby potestować działanie to sprawdzę i ewentualnie podmienię :)

Reply:
Poprawię na Twoją propozycję, wydajnościowo w naszym przypadku widocznej różnicy nie ma, ale może kiedyś w aplikacji będzie milion rekordów i wtedy to się przyda :)